medcat/cat.py

```diff
@@ -422,11 +417,9 @@ def load_model_pack(cls,
             addl_ner.append(trf)
 
         # Find metacat models in the model_pack
-        meta_paths = [os.path.join(model_pack_path, path) for path in os.listdir(model_pack_path) if path.startswith('meta_')] if load_meta_models else []
         meta_cats = []
-        for meta_path in meta_paths:
-            meta_cats.append(MetaCAT.load(save_dir_path=meta_path,
-                                          config_dict=meta_cat_config_dict))
+        if load_meta_models:
+            meta_cats = cls.load_meta_cats(model_pack_path)
```

Comment:
Same overall comment as above.

So I think the easiest solution to the linting issue would be to type-annotate here, e.g `meta_cats: List[MetaCAT] = cls.load_meta_cats(model_pack_path)`.

Reply:
done